src/coreclr/jit/llvm.cpp

```diff
@@ -1639,6 +1645,65 @@ unsigned Llvm::buildMemCpy(Value* baseAddress, unsigned startOffset, unsigned en
     return size;
 }
 
+void Llvm::buildThrowException(llvm::IRBuilder<>& builder, const char* helperClass, const char * helperMethodName, Value* shadowStack)
+{
+    CORINFO_METHOD_HANDLE methodHandle = _getCompilerHelpersMethodHandle(_thisPtr, helperClass, strlen(helperClass),
```

Comment:
The JIT/EE interface typically sends the strings over as zero terminated strings and then constructs the managed string using `Marshal.PtrToStringUTF8`. Fewer arguments to pass through.

Reply:
Thanks, I copied
```
private int doAssert(byte* szFile, int iLine, byte* szExpr)
        {
            Log.WriteLine(Marshal.PtrToStringAnsi((IntPtr)szFile) + ":" + iLine);
            Log.WriteLine(Marshal.PtrToStringAnsi((IntPtr)szExpr));

            return 1;
        }
```